uvtools/dspec.py

```diff
@@ -124,17 +471,31 @@ def high_pass_fourier_filter(data, wgts, filter_size, real_delta, clean2d=False,
              This allows us to avoid the problem that if we have RFI flagging and apply a linear filter
              that is larger then the horizon then the foregrounds that we fit might actually include super
              -horizon flagging side-lobes and restoring them will introduce spurious structure.
+        fg_deconv_fundamental_period: int, optional
+            fundamental period of Fourier modes to fit too.
```

Comment:
not sure if I understand this based on the description. can you elaborate?

Reply:
if you are performing dft_interpolation, then fg_deconv_fundamental_period specifies the wavelength in the denominator of the fourier mode exponent (sets the N in e^{-2 \pi j n m / N} )

Reply:
got it